jvcl/run/JvDriveCtrls.pas

```diff
@@ -1652,6 +1662,21 @@ procedure TJvFileListBox.ReadFileNames;
   end;
 end;
 
+procedure TJvFileListBox.SetAutoUpdate(Value: Boolean);
+begin
+  if Value <> FAutoUpdate then
+  begin
+    // Lazy initialization of JvChangeNofity
+    if FAutoUpdate = false then
```

Comment:
Please never test with `= false`, it's very misleading.
The proper way to write this test is `if not FAutoUpdate then`